src/mscorlib/ref/mscorlib.cs

```diff
@@ -8778,6 +8778,8 @@ public enum FlowControl
         Cond_Branch = 3,
         Meta = 4,
         Next = 5,
```

Comment:
BTW: src/mscorlib/ref/mscorlib.cs is not really used for much anymore. It is not necessary to update it to expose the API, but it does not hurt if you do it.